test/OpenTelemetry.AutoInstrumentation.Tests/Configuration/SettingsTests.cs

```diff
@@ -47,7 +47,7 @@ public void Dispose()
     }
 
     [Fact]
-    public void TracerSettings_DefaultValues()
+    internal void TracerSettings_DefaultValues()
```

Comment:
It's not a recommended approach to add the public API based change to test project. OTel SDK does not modify test projects.

Reply:
It was due to Traces/Metrics/LogsExporter. It is internal, so the method cannot be public. I know that I can probably change it to few public methods, but I hope this solution is acceptable.

See:
 `internal void TracesExporter_SupportedValues(string tracesExporter, TracesExporter expectedTracesExporter)`

Reply:
It's an additional overhead to maintain these for test projects. I don't have strong preference.